packages/core/package.json

```diff
@@ -60,14 +60,12 @@
     ".": {
       "import": {
         "types": "./dist/type/index.d.ts",
+        "edge-light": "./dist/index.edge.js",
```

Comment:
@himself65 the idea is that `@llamaindex/edge` is working with edge, but `llamaindex` is not - that's why i removed `edge-light` from the core package